tools/who_what_benchmark/whowhatbench/wwb.py

```diff
@@ -42,6 +44,8 @@ def __init__(self, model, model_dir, model_type):
         self.model = model
         self.model_type = model_type
 
+        if model_type == "text" or model_type == "visual-text":
+            self.config = AutoConfig.from_pretrained(model_dir, trust_remote_code=True)
         if model_type == "text":
             self.config = AutoConfig.from_pretrained(
                 model_dir, trust_remote_code=True)
```

Comment:
looks like at this moment you already have loaded config according to condition above and you load it twice here

Reply:
merge issue, fixed